plugins/flatpak/gs-flatpak.c

```diff
@@ -594,7 +594,8 @@ gs_flatpak_add_apps_from_xremote (GsFlatpak *self,
 
 		/* Mark com.endlessm. apps with GnomeSoftware::popular kudo. See discussion on T23152 */
 		if (!as_app_has_kudo (app, "GnomeSoftware::popular") &&
-		     g_str_has_prefix (as_app_get_id (app), "com.endlessm."))
+		     g_str_has_prefix (as_app_get_id (app), "com.endlessm.") &&
+		     g_strcmp0 (as_app_get_id (app), "com.endlessm.HatchPreviewer") != 0)
```

Comment:
I know it might be a little hair-splitting but my gut feeling is this is more reliably expressed as all com.endlessm apps from eos-apps should be auto-featured, and perhaps we also wish to feature (from flathub) com.endlessm.photos?

Reply:
I have implemented this - com.endlessm is now only matched on *.endlessm.com origins.